### PR TITLE
GLT-3107: restrict approving run data to a group

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Changes:
 
+  * Added "Run Approvers" group. Only users in this group will be allowed to modifiy "Data
```
Comment (callunity) [inline RELEASE_NOTES.md]: `modifiy` -> `modify`

File: miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java [hunk 2]
```diff
@@ -355,6 +356,10 @@ private void validateChanges(Run before, Run changed) throws IOException {
       errors.add(new ValidationError("sequencingKitId", "Must be a sequencing kit"));
     }
 
+    if (((before == null && changed.isDataApproved() != null) || isChanged(Run::isDataApproved, changed, before))
+        && authorizationManager.getCurrentUser().getGroups().stream().noneMatch(group -> Group.RUN_APPROVERS.equals(group.getName()))) {
```
Comment (apmasell) [inline miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java]: or admin? I guess this is a policy question and not a technical one...

File: miso-web/src/main/webapp/scripts/form_run.js [hunk 3]
```diff
@@ -231,6 +233,20 @@ FormTarget.run = (function($) {
...
+          getDisplayValue: function(item) {
+            if (item.dataApproved === true) {
+              return 'Yes';
+            } else if (item.dataApproved === false) {
+              return 'No';
+            } else {
+              return 'Unknown';
+            }
+          },
+          include: !config.isRunApprover
```
Comment (apmasell) [inline miso-web/src/main/webapp/scripts/form_run.js]: or admin?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (djcooke) [reply RELEASE_NOTES.md]: fixed

File: miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java (hunk 2 above)
Comment (djcooke) [reply miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java]: :+1: Changed so admin can approve runs too

File: miso-web/src/main/webapp/scripts/form_run.js (hunk 3 above)
Comment (djcooke) [reply miso-web/src/main/webapp/scripts/form_run.js]: :+1: changed

**Pull request merged**